birdhouse/components/thredds/default.env

```diff
@@ -74,5 +74,4 @@ export DELAYED_EVAL="
   THREDDS_SERVICE_DATA_LOCATION_ON_HOST
   THREDDS_IMAGE
   THREDDS_IMAGE_URI
-  THREDDS_DATASET_DATASETSCAN_BODY
```

Comment:
Note to reviewers: This cannot be a delayed eval variable because of the way that the delayed evaluation process removes quotes. If we change the delayed eval process to allow this, it will break other functionality (especially in jupyterhub).